### PR TITLE
doc: add TokenPattern description

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: doc/locale/ja/LC_MESSAGES/reference.po [hunk 1]
```diff
@@ -28954,6 +28954,68 @@ msgstr ""
...
+msgstr ""
+"``TokenPattern`` の例です。``TokenPattern`` は指定した正規表現にマッチする"
+"トークンのみを抽出します。そのためホワイトリストみたいなことができます。"
+
+msgid ""
+"For example, let's hide a specific keyword from search results even though "
+"search query contains it."
+msgstr ""
+"例えば、検索するクエリーに含まれていても、特定のキーワードは検索結果から隠す"
+"というのをやってみましょう。"
```
Comment (komainu8) [inline doc/locale/ja/LC_MESSAGES/reference.po]: In this case, "TokenPattern" looks like that used as blacklists.

Comment (kenhys) [reply doc/locale/ja/LC_MESSAGES/reference.po]: I've fixed description.

**Comment by kenhys** [comment]
@komainu8 could you review again?

File: doc/locale/ja/LC_MESSAGES/reference.po (hunk 1 above)
Comment (komainu8) [reply doc/locale/ja/LC_MESSAGES/reference.po]: Ah...I'm sorry.

In before example, the description was as a whitelist but the example was as a blacklist.
Therefore, in my opinion, we should align the example and description.

**Comment by kenhys** [comment]
@komainu8 Did you review based on old version?
I've rewritten that explanation, so it should be removed already.
(Added one more sample and explain hit/not hit test case)

**Pull request merged**